File: xssfinder-java-executor/src/main/java/org/xssfinder/runner/TraversalResult.java
package org.xssfinder.runner;

import java.util.Map;

public class TraversalResult {
    private Object page;
    private Map<String, String> inputIdsToAttackIds;

    public TraversalResult(Object page, Map<String, String> inputIdsToAttackIds) {
        this.page = page;
        this.inputIdsToAttackIds = inputIdsToAttackIds;
    }

    public Object getPage() {
        return page;
    }

    public Map<String, String> getInputIdsToAttackIds() {
        return inputIdsToAttackIds;
    }
}

File: xssfinder-php-executor/src/XssFinder/Runner/TraversalResult.php
<?php

namespace XssFinder\Runner;

class TraversalResult
{
    private $_page;
    private $_inputIdsToAttackIds;

    public function __construct($page, array $inputIdsToAttackIds)
    {
        $this->_page = $page;
        $this->_inputIdsToAttackIds = $inputIdsToAttackIds;
    }

    public function getInputIdsToAttackIds()
    {
        return $this->_inputIdsToAttackIds;
    }

    public function getPage()
    {
        return $this->_page;
    }
}